src/pages/Home.jsx

```diff
@@ -31,84 +51,64 @@ export default function Home() {
-        body="Codify is a newly registered UC Berkeley student organization
-              focused on bringing students together to work on impactful
-              software projects. Codify is an opportunity to develop new skills,
-              gain real world development experience, and find a community of
-              like minded students."
-        link="/AboutUS"
+        body="Codify is a recently established student organization at UC Berkeley focused on bringing students together to work on impactful software projects. Our mission is to provide students with the opportunity to develop new skills, gain real-world development experience, and connect with a community of like-minded peers."
+        link="/About"
         linkText="About Our Team and Mission"
       ></Card>
       <Card
         imageSource={require("../assets/Images/General/Elaine.png")}
         header="Building Exciting Things..."
-        body="Every semester, Codify works on projects in web development, data
-        analysis, mobile development, and any other field of computing
-        that our members are passionate about. We maintain a healthy mix
-        of developing projects for both nonprofit and for profit clients,
-        and launching our own internal projects to either develop into a
-        startup or release for the good of the Berkeley community."
+        body="Every semester, Codify works on projects in web development, data analysis, mobile development, and other fields of computing that our members are passionate about. We uniquely focus on developing projects for startups and launching our own internal projects for the good of the Berkeley community."
         link="/Projects"
-        linkText="What we've been working on"
+        linkText="Current work"
       ></Card>
       <Card
         imageSource={require("../assets/Images/General/gate.jpeg")}
         header="Within the Berkeley Community."
-        body="Codify is an organization created by and for the Berkeley computer
-        science community. We strive to provide opportunities for students
-        to find their place in tech, explore their entrepreneurial
-        aspirations, and use their skills to give back to their
-        communities. We are always looking for passionate students to join
-        our ranks, and partner organizations to work with towards our
-        mutual goals."
+        body="Codify is an organization created by and for the Berkeley computer science community. We strive to provide opportunities for students to find their place in tech, explore their entrepreneurial aspirations, and use their skills to give back to their communities. We are always looking for passionate students to join our ranks and partner with organizations to work toward our mutual goals."
         link="/WorkWithUs"
-        linkText="Work With Codify"
+        linkText="Collaborate with us"
       ></Card>
     </div>
   );
 }
 
 function Card(props) {
```

Comment:
This should have been done earlier, but can you move this to be it's own component in the components folder?

Reply:
What is the "this" you're referring to?